utils/python/CIME/case_run.py

```diff
@@ -171,7 +171,8 @@ def resubmit_check(case):
     elif dout_s and mach == 'mira':
         caseroot = case.get_value("CASEROOT")
         cimeroot = case.get_value("CIMEROOT")
-        cmd = "ssh cooleylogin1 'cd %s; CIMEROOT=%s ./case.submit %s --job case.st_archive' "%(caseroot, cimeroot, caseroot)
+        cmd = "ssh cooleylogin1 'cd %s; CIMEROOT=%s ./case.submit %s"%(caseroot, cimeroot, caseroot)
+        cmd = cmd + " --job case.st_archive' "
```

Comment:
I just run pylint (1.6.4). It complained that the line was too long. I tried some simple line breaks but it complained so I broke it up this way :(
I just realized that I forgot the -C, I think I can put that back the way it was.